pkg/apis/cluster/v1alpha1/common_cluster_status_types.go

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
+
+// StatusCluster is shared type to contain provider independent cluster status
+// information.
+type StatusCluster struct {
```

Comment:
Can we name the type according to the file? I am not happy with the naming as we have it for the old types. It is super confusing because there is no symmetry whatsoever. 

```suggestion
type CommonClusterStatus struct {
```